pgmpy/inference/ExactInference.py

```diff
@@ -171,24 +172,25 @@ def _variable_elimination(
             # Removing all the factors containing the variables which are
             # eliminated (as all the factors should be considered only once)
             factors = [
-                factor
+                factor[0]
                 for factor in working_factors[var]
```

Comment:
Same here.

Reply:
Yes, here we can switch to "factor, _". I have done it. Should I issue a new PR, along with the new test?

Reply:
You can just commit and push to your branch and it should automatically update this PR.